.github/generate_dev_tests.py

```diff
@@ -0,0 +1,49 @@
+import shutil
+from pathlib import Path
+
+import yaml
+
+tests = [
+    "test_exporters.yml",
+    "test_dummy_inputs.yml",
+    "test_bettertransformer.yml",
+    "test_onnx.yml",
+    "test_fx.yml",
+    "test_onnxruntime.yml",
+    "test_benckmark.yml",
+    "test_optimum_common.yml",
+]
+
+for test_name in tests:
+    new_name = "dev_" + test_name
+
+    with open(Path("workflows", test_name), "r") as file:
+        workflox_yml = yaml.load(file, yaml.BaseLoader)
+
+        workflox_yml["name"] = "dev_" + workflox_yml["name"]
+        workflox_yml["on"] = {"schedule": [{"cron": "0 7 * * *"}]}
+
+        for i, step in enumerate(workflox_yml["jobs"]["build"]["steps"]):
+            if "name" in step and step["name"] == "Install dependencies":
+                workflox_yml["jobs"]["build"]["steps"][i][
+                    "run"
+                ] += "pip install -U git+https://github.com/huggingface/transformers@main#egg=evaluate\npip install -U git+https://github.com/huggingface/transformers@main#egg=diffusers\npip install -U git+https://github.com/huggingface/transformers@main#egg=transformers\n"
```

Comment:
Why do we install packages this way with the `#egg`? Not sure what it does exactly (it seems useful for installing extras, but that's not the case here).
I would have expected:
```bash
pip install -U git+https://github.com/huggingface/transformers
pip install -U git+https://github.com/huggingface/diffusers
pip install -U git+https://github.com/huggingface/evaluate
```

Reply:
I just copy pasted, thanks!